app/controllers/state_file/faq_controller.rb

```diff
@@ -4,11 +4,9 @@ class StateFile::FaqController < ApplicationController
   def index
     @state_code_names = case params[:us_state]
                         when 'us'
```

Comment:
Maybe this would make more sense as an index/show pair of methods, and we drop the 'us' param when showing the whole page? I.e. there's a new route with no us_state param that goes to index, and index just returns the state_code_to_name_map, and a route with a us_state_param that goes to show, and there we do the slice?

Reply:
^ this is out of scope for this change, just havin' a ponder

Reply:
Maybe ditch the case parameter here too in favor of an if else?